tests/conftest.py

```diff
@@ -195,9 +195,17 @@ def test_pd_df():
     yield TEST_DF.copy()
 
 
-@pytest.fixture(scope="function")
-def meta_df():
-    df = IamDataFrame(data=TEST_DF)
+@pytest.fixture(
+    scope="function",
+    params=[
+        [2005, 2010],
+        [datetime(2005, 6, 17), datetime(2010, 7, 21)],
```

Comment:
this list of `datetimes` is used quite a lot now throughout the tests - maybe make that available as static object and reuse throughout?

Reply:
nice will do!